app/src/main/java/com/example/dine_and_donate/HomeFragments/MapFragment.java

```diff
@@ -448,22 +434,48 @@ public void onCancelled(DatabaseError databaseError) {
         });
     }
 
-    private void slideUpAnimation(final JSONObject restaurant) throws JSONException {
-        TextView restName = slideView.findViewById(R.id.tv_restaurant_name);
-        restName.setText(restaurant.getString("name"));
-        slideView.setVisibility(View.VISIBLE);
-        TranslateAnimation animate = new TranslateAnimation(
-                0,
-                0,
-                slideView.getY(),
-                0);
-        animate.setDuration(500);
-        animate.setFillAfter(true);
-        slideView.startAnimation(animate);
-        btnCreate = slideView.findViewById(R.id.btn_create_event);
+    private boolean clickMarker(Marker marker) {
+        try {
+            final JSONObject restaurantOfMarker = (JSONObject) marker.getTag();
+            System.out.println(restaurantOfMarker);
```

Comment:
i'd recommend deleting printlns before pushing to master, but up to yall! @ryan-beck, @shry678